### ISSUE TITLE
2552-9 flat structure

### ISSUE THREAD

**Issue by thoffeller** [CAFM-Connect/BIM-Profiles#2]
Hi Klaus,
the bim-profile for sheet 9 is flat, no tree in the classification.
maybe an export issue?

Cheers,
Tim

**Comment by klacol** [comment]
Ok, I wil have a look at it

**Comment by thoffeller** [comment]
With the latest commit the 2552-9 guideline is still flat
![grafik](https://user-images.githubusercontent.com/8693405/46442319-8adc7f80-c769-11e8-9048-d4811ce5691a.png)

**Comment by thoffeller** [comment]
a lot of relations are missing (have a look at the scrollbar on the right)

**Comment by klacol** [comment]
Yes, I know. The ifcXML file of the profile does not reflect yet the IfcFirst strategy. I have to work on it. Good to know, that you have allready a profile loader that is able to consumes the profiles.

**Comment by thoffeller** [comment]
Does it depend on the ifc first or parent relations? If you can fix this, I'll add my extensions to the MEP part directly to 2552-9 instead of BTGA profile?
